_posts/en/newsletters/2020-11-04-newsletter.md

```diff
@@ -0,0 +1,113 @@
+    generated.
+
+- **Taproot activation survey results:** Anthony Towns received survey
+  responses from an unnamed set of 13 "smart dev-type people" about
+  their preferences for a soft fork activation mechanism to use with
+  taproot.  The [post][towns survey] includes his own personal summary
+  of the results, which we quote in abbreviated form below:
+
+    > - Activation threshold should stay at 95% [...]
+    >
+    > - [...] hope for a quick response [from miners] within a few
+    >   months, but plan for it taking up to a year, even if everything
+    >   goes well
+    >
+    > - Setting the [earliest possible activation time] to be about a
+    >   month or two [after the Bitcoin Core release containing the
+    >   activation code] is probably about right [...]
+    >
+    > - Almost everyone is open to the idea of a flag day in some
+    >   circumstances
+    >
+    > - If there’s a flag day, we should expect it to be a year or two
+    >   away [...]
+    >
+    > - There probably isn’t support for setting a flag day initially
+    >   [...]
+    >
+    > - Almost everyone wants to see as many nodes as possible enforce
+    >   the rules after activation
+    >
+    > - Most people seem to be willing to accept bringing a flag day
+    >   forward by mandatory signalling
+    >
+    > - There’s not a lot of support for opting-in to flag day
+    >   activation by setting a configuration option
+
+    After Bitcoin Core developers finish working on the upcoming 0.21
+    release, it's likely that the results of this survey will help them
+    choose the activation parameters that will be used for taproot in a
+    subsequent release.  Of course, actual enforcement of the soft fork
+    will depend on it gaining broad support among users---and users may
+    decide to use an alternative activation mechanism.  Further
+    discussion of activation parameters is encouraged in the
+    [##taproot-activation][] IRC chatroom ([logs][##taproot-activation
+    logs]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #4162][] updates logging to include whether a payment used
+  a [payment secret][]. Payment secrets were added to BOLT 11 in [BOLTs #643][]
+  as part of the [multipath payments][topic multipath payments] feature, and prevent probing attacks by nodes
+  along the payment path. Payment secrets are supported by all LN
+  implementations and will eventually be made compulsory by C-Lightning; logging
+  which payments are using payment secrets makes it easier for the developers to
+  know when it's reasonable to make that change.
```

Comment:
@jnewbery I s/safe/reasonable/ because I'm unaware of any safety-related risks to requiring payment secrets.  I think the only risk to requiring them is not being able to pay old-style invoices that don't included them.  As always, please feel free to revert.

Reply:
This seems like a perfectly _reasonable_ change to me.